### PR TITLE
Test for Get Object with Preview

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: integration/objects_test.go [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+		"myobject.jpg", fileReader, int64(len(fakeFile)), minio.PutObjectOptions{ContentType: "application/octet-stream"})
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	assert := assert.New(t)
+	type args struct {
+		encodedPrefix string
+		versionId     string
```
Comment (kaankabalak) [inline integration/objects_test.go:80]: @dvaldivia, I think you might've already seen this, but there seems to be an error with the variable name here:

```suggestion
		versionID     string
```

File: integration/objects_test.go [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			client := &http.Client{
+				Timeout: 3 * time.Second,
+			}
+			destination := fmt.Sprintf("/api/v1/buckets/%s/objects/download?preview=true&prefix=%s&version_id=%s", bucketName, tt.args.encodedPrefix, tt.args.versionId)
+			finalUrl := fmt.Sprintf("http://localhost:9090%s", destination)
```
Comment (kaankabalak) [inline integration/objects_test.go:175]: @dvaldivia, I think you might've already seen this too, but GitHub once again seems to be complaining with an issue with the variable name here:
```suggestion
			finalURL := fmt.Sprintf("http://localhost:9090%s", destination)
```

File: integration/objects_test.go [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+			request, err := http.NewRequest("GET", finalUrl, nil)
```
Comment (kaankabalak) [inline integration/objects_test.go:176]: Similarly, if we are to replace `finalUrl` with `finalURL` as GitHub suggested, we would also need to do this:

```suggestion
			request, err := http.NewRequest("GET", finalURL, nil)
```

File: integration/objects_test.go (hunk 2 above)
Comment (dvaldivia) [reply integration/objects_test.go]: that's right, I've updated the PR.

File: integration/objects_test.go (hunk 3 above)
Comment (dvaldivia) [reply integration/objects_test.go]: that's right, I've updated the PR.

File: integration/objects_test.go (hunk 1 above)
Comment (dvaldivia) [reply integration/objects_test.go]: that's right, I've updated the PR.

**Review by jinapurapu — DISMISSED** [review]

**Review by cniackz — APPROVED** [review]
Looks good to me, added the increment:
```
-          threshold=35.60
+          threshold=36.5
```
Thank you Daniel for your help

**Comment by cniackz** [comment]
And we are getting:
```
Error: buildx call failed with: error: failed to solve: unexpected status: 403 Forbidden
```
Reading: https://github.community/t/403-error-on-container-registry-push-from-github-action/173071/18

**Review by bexsoft — APPROVED** [review]

**Pull request merged**